### PR TITLE
[BREAKING] Octane upgrade v3.12.0...v3.16.2

### PR DESCRIPTION
**Pull request by rajasegar**
Fixes errors with the latest  class syntax for modern Ember apps.

Related #220 #240

### CODE REVIEW

**Comment by rajasegar-c** [comment]
Related #220 #240

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     "babel-plugin-transform-async-to-generator": "^6.24.1",
     "body-parser": "^1.19.0",
     "co": "^4.6.0",
-    "ember-cli-babel": "^7.11.0",
+    "ember-cli-babel": "^7.17.2",
+    "ember-cli-htmlbars": "^4.2.2",
```
Comment (rwjblue) [inline package.json:35]: Can you move this back into devDependencies? I do not believe that it is needed as a dep.

File: package.json [hunk 2]
```diff
@@ -42,43 +43,48 @@
...
     "loader.js": "^4.7.0",
     "mocha": "^5.2.0",
-    "qunit-dom": "^0.8.4",
+    "qunit-dom": "^1.0.0",
     "rimraf": "^2.6.2",
     "sinon": "^4.2.2"
   },
   "engines": {
-    "node": "8.* || >= 10.*"
+    "node": "10.* || >= 12"
```
Comment (rwjblue) [inline package.json:84]: Just pointing out, this needs to be a breaking change release

**Comment by rwjblue** [comment]
> Fixes errors with the latest class syntax for modern Ember apps.

What specifically is fixing these errors? The ember-cli-babel update?

File: package.json (hunk 1 above)
Comment (rajasegar) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (rajasegar) [reply package.json:84]: Thanks Robert, will update the PR subject for the same.

**Comment by rajasegar** [comment]
> > Fixes errors with the latest class syntax for modern Ember apps.
> 
> What specifically is fixing these errors? The ember-cli-babel update?

Yes, @rwjblue there were some errors related to class syntax I remember, but I am not quite sure about that If I use the 1.0 latest beta, it was working fine. Anyways I thought for compatibility reasons it's better to be on the latest LTS

**Comment by rwjblue** [comment]
>  Anyways I thought for compatibility reasons it's better to be on the latest LTS

Totally agree!!!

**Pull request merged**